src/dsl/step.ts

```diff
@@ -30,16 +30,19 @@ export class Step {
   index: number;
   callback: VoidCallback;
   location?: Location;
+  isSkipped = false;
```

Comment:
would be simpler to keep this as static variable and use it from the addStep. We might add more mofidiers/annotations later which might increase the args length.